field_audit/management/commands/bootstrap.py

```diff
@@ -0,0 +1,141 @@
+from contextlib import contextmanager
```

Comment:
Suggestion: rename this management command since management commands in Django share a global namespace.

Reply:
Renamed in b3778b2.